src/workerd/jsg/modules.c++

```diff
@@ -279,9 +279,9 @@ void instantiateModule(jsg::Lock& js, v8::Local<v8::Module>& module) {
   KJ_ASSERT(!module.IsEmpty());
   auto isolate = js.v8Isolate;
   auto context = isolate->GetCurrentContext();
+  js.tryCatch([&] {
```

Comment:
nit: indent below. Or did you intend to remove this tryCatch?

Reply:
intended to remove. that was missed cleanup